plugins/examples/security-authorization-engine/src/javaRestTest/java/org/elasticsearch/example/CustomAuthorizationEngineIT.java

```diff
@@ -50,9 +47,13 @@ protected Settings restClientSettings() {
     }
 
     public void testClusterAction() throws IOException {
-        RestHighLevelClient restClient = new TestRestHighLevelClient();
-        restClient.security().putUser(PutUserRequest.withPassword(new User("custom_user", List.of("custom_superuser")),
-            "x-pack-test-password".toCharArray(), true, RefreshPolicy.IMMEDIATE), RequestOptions.DEFAULT);
+        ElasticsearchClient restClient = new ElasticsearchClient(new RestClientTransport(client(), new JacksonJsonpMapper()));
+        restClient.security().putUser(req -> req
+            .username("custom_user")
```

Comment:
Is there no equivalent for the refresh options here? That said, I don't know if they're necessary.

Reply:
"Immediate" is the default.